lib/JSON/CalDAV/CalendarHandler.php

```
@@ -0,0 +1,476 @@
+            }
+        }
+
+        $sharingPlugin->shareResource($path, $sharees);
+
+        // see vendor/sabre/dav/lib/CalDAV/SharingPlugin.php:268
+        $this->server->httpResponse->setHeader('X-Sabre-Status', 'everything-went-well');
+
+        return [200, null];
+    }
+
+    public function updateInviteStatus($path, $node, $jsonData) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$path'. (undefined)

(UnusedFormalParameter)